psc-package/Main.hs

```diff
@@ -128,6 +129,9 @@ readPackageSet PackageConfig{ set } = do
       exit (ExitFailure 1)
     Just db -> return db
 
+getGitRepoList :: PackageConfig -> [(PackageName, (Repo, Version))]
+getGitRepoList = toListOf ((ifolded <. to (repo &&& version)) . withIndex)
```

Comment:
This was some code golf while I was working on this originally. I think here, I'd prefer the simpler, lens-free version.